x/zoneconcierge/README.md

```diff
@@ -0,0 +1,500 @@
+checkpoint headers of PoS blockchains to Bitcoin, the Zone Concierge module
+builds an index recording headers' positions on Babylon's ledger, which will
+eventually be checkpointed by Bitcoin. To this end, the Zone Concierge module
+intercepts headers from IBC light clients via a
+[PostHandler](https://docs.cosmos.network/v0.50/learn/advanced/baseapp#runtx-antehandler-runmsgs-posthandler),
+and indexes them.
+
+Note that the Zone Concierge module intercepts all headers that have a valid
+quorum certificate, including canonical headers and fork headers. A fork header
+with a valid quorum certificate is a signal of the dishonest majority attack:
+the majority of validators are dishonest and sign conflicted headers.
+
+**Providing Proofs that a Header is Checkpointed by Bitcoin.** To support use
+cases that need to verify BTC timestamps of headers, Zone Concierge can provide
+proofs that the headers are indeed checkpointed to Bitcoin. The proof includes
+the following:
+
+- `ProofCzHeaderInEpoch`: Proof that the header of the PoS blockchain is
+  included in an epoch of Babylon;
+- `ProofEpochSealed`: Proof that the epoch has been agreed by > 2/3 voting power
+  of the validator set; and
+- `ProofEpochSubmitted`: Proof that the epoch's checkpoint has been submitted to
+  Bitcoin.
+
+The first proof is formed as a Merkle proof that the IBC header is committed to
+the `AppHash` after the epoch. The second proof is formed as a BLS
+multi-signature jointly generated by the epoch's validator set. The last proof
+is formed as Merkle proofs of two transactions that constitute a BTC checkpoint,
+same as in [BTCCheckpoint module](../btccheckpoint/README.md).
+
+### Use cases
+
+The Bitcoin-checkpointed PoS blockchain will enable several applications, such
+as raising alarms upon dishonest majority attacks and reducing the unbonding
+period. These use cases require new plugins in the PoS blockchains, and will be
+developed by Babylon team in the future.
+
+**Raising Alarms upon Dishonest Majority Attacks.** Zone Concierge timestamps
+fork headers that have valid quorum certificates. Such fork header signals a
+safety attack launched by the dishonest majority of validators. Babylon can send
+the fork header back to the corresponding PoS blockchain, such that the PoS
+blockchain will get notified with this dishonest majority attack, and can decide
+to stall or initiate a social consensus.
+
+**Reducing Unbonding Period.** Zone Concierge provides a Bitcoin-checkpointed
+prefix for a PoS blockchain. Such Bitcoin-checkpointed prefix resists against
+the long range attacks, thus unbonding requests in this prefix can be safely
+finished, leading to much shorter unbonding period compared to that in existing
+PoS blockchains (e.g., 21 days in Cosmos SDK chains).
+
+## State
+
+The Zone Concierge module keeps handling IBC headers of PoS blockchains, and
+maintains the following KV stores.
+
+### Parameters
```

Comment:
structures are out of date with the protos

Reply:
captured in https://github.com/babylonlabs-io/babylon/issues/428